File: balancer/service/gate/src/core/Seq.cc
#include "Seq.h"
#include <ctime>
#include <cstring>

#include "tool/Util.h"

Seq::Seq() : count(0)
{

}

Seq::~Seq()
{

}

unsigned long long Seq::make_seq()
{
	if(false)
	{
		return Util::uiui2ull(::time(nullptr), ++count);
	}
	else
	{
		return ++count;
	}
}

File: balancer/service/proxy/src/core/seq.py
# -*- coding: utf-8 -*-

from src.tool import util


class Seq:

    EN_SEQ_NULL = 0

    def __init__(self):
        self.count = 0

    def make_seq(self):
        self.count += 1
        if False:
            return util.uiui2ull(util.now_s(), self.count)
        else:
            return self.count


